examples/avalon/plugins/RoleInfos.ts

```diff
@@ -44,3 +46,5 @@ export default Vue.extend({
     },
   },
 });
+
+window.customElements.define("role-infos", wrap(Vue, Component));
```

Comment:
i don't like this part... it's weird that both the name of the file and name of the component matter (and you have to to make sure the component name is the kebab case version of the file name)